mainwindow.cpp

```diff
@@ -157,17 +157,17 @@ MainWindow::MainWindow(QSettings &settings)
     // TODO: Are this checks are enough?
     // TODO: Claim on mkdir fails.
     // TODO: Do not do it on start up. Choose other good time.
-    if (!QDir(QDir::home().filePath(".john")).exists()) {
-        QDir::home().mkdir(".john");
+    if (!QDir(QDir::home().filePath("john")).exists()) {
```

Comment:
Removinh the . in .john is temporary, it's until we get a final decision from Solar because right now dots no longer works with cvs repo version